### PR TITLE
(bugsbash) fixes error return values is not checked

### PR DESCRIPTION
**Pull request by asquare14**
Signed-off-by: aSquare14 <atibhi.a@gmail.com>

### CODE REVIEW

File: local/local_test.go [hunk 1]
```diff
@@ -71,7 +71,11 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 				h.AssertNil(t, err)
 				h.AssertNil(t, img.Save())
 
-				defer h.DockerRmi(dockerClient, img.Name())
+				defer func() {
+					err = h.DockerRmi(dockerClient, img.Name())
+					h.AssertNil(t, err)
+
```
Comment (natalieparellano) [inline local/local_test.go:77]:
```suggestion
```

File: local/local_test.go [hunk 2]
```diff
@@ -106,7 +110,11 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
...
```
Comment (natalieparellano) [inline local/local_test.go:116]:
```suggestion
```

File: local/local_test.go [hunk 3]
```diff
@@ -191,7 +199,11 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 						)
 						h.AssertNil(t, err)
 						h.AssertNil(t, img.Save())
-						defer h.DockerRmi(dockerClient, img.Name())
+						defer func() {
+							err = h.DockerRmi(dockerClient, img.Name())
+							h.AssertNil(t, err)
+
```
Comment (natalieparellano) [inline local/local_test.go:205]:
```suggestion
```

File: local/local_test.go (hunk 1 above)
Comment (natalieparellano) [reply local/local_test.go:77]: (to make the linter happy)

**Comment by asquare14** [comment]
@natalieparellano Thank you for the review ! Done :D

**Review by natalieparellano — APPROVED** [review]

**Pull request merged**